motor_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: spin_joint
    joint_speed: 11.0
    motor:
        id: 8
        init: 0
        min: 0
        max: 1023
